crates/wasm-encoder/Cargo.toml

```diff
@@ -33,8 +33,10 @@ wasmparser = { path = "../wasmparser" }
 wasmprinter = { workspace = true }
 
 [features]
-default = ['component-model']
+default = ['component-model', 'simd']
 
 # On-by-default: conditional support for emitting components in addition to
 # core modules.
 component-model = ['wasmparser?/component-model']
+# On-by-default: conditional support for emitting SIMD wasm operators.
+simd = ['wasmparser?/simd']
```

Comment:
Given that this feature only gates the wasmparser-to-wasm-encoder conversion, as opposed to gating the instructions themselves, I think it might be better here to just unconditionally enable the `simd` feature of wasmparser. That can be left as future work to implement full conditional gating in `wasm-encoder`

Reply:
I agree. I did this just because I wanted to mic the behavior for the component-model. Enabling sims by default should also decrease the diff of this PR which I welcome.